lisa/util/__init__.py

```diff
@@ -3,6 +3,9 @@
 
 import re
 from datetime import datetime
+from lisa.util import constants
+from lisa.secret import replace
```

Comment:
Import it like below, so it's clear that the replace method is from secret module.

`from lisa import secret`